src/EnergyPlus/HeatBalanceMovableInsulation.cc

```diff
@@ -177,8 +179,12 @@ namespace HeatBalanceMovableInsulation {
 
 			HMovInsul = 1.0 / ( MovInsulSchedVal * Material( Surface( SurfNum ).MaterialMovInsulExt ).Resistance );
 			RoughIndexMovInsul = Material( Surface( SurfNum ).MaterialMovInsulExt ).Roughness;
-			AbsExt = max( 0.0, 1.0 - Material( Surface( SurfNum ).MaterialMovInsulExt ).Trans - Material( Surface( SurfNum ).MaterialMovInsulExt ).ReflectSolBeamFront );
-
+			MaterialGroupNum = Material( Surface( SurfNum ).MaterialMovInsulExt ).Group;
```

Comment:
Minor comments:
 - Consider defining `MaterialGroupNum` directly where it is used, no need to define variables at the top of each function like we had to do with FORTRAN.  Making that a habit can ensure the variable is scoped properly, which can aid in the compiler auto-optimizing the code.
 - Consider cleaning up empty and unnecessary comment block sections from the old FORTRAN template (`DERIVED TYPE DEFINITIONS`, etc.) as you go, to slowly clean the code up.
 - There are a number of lookups for the `Surface( SurfNum ).MaterialMovInsulExt`.  The compiler _may_ be optimizing these away, but it may be better to reduce these calls down by using a single reference.  Add in `auto & thisSurface( Surface( SurfNum ) )`, and then change each of those `Surface( SurfNum )` calls to just be `thisSurface`. Like I said, it may not affect the compiler, but it will make a lighter looking block of code.